azure_media_services/templates/player.html

```diff
@@ -49,10 +49,6 @@
                 </select>
               </div>
             </div>
-            <div class="embed-tou">
```

Comment:
Its not very clear from the PR description and the acceptance criteria if this text that is removed has to be replaced with new text?

Reply:
Acceptance Criteria mentions that:
"When clicking the Share button from the video player, this section is completely remove" The player is governed by this license. Please read it. By copying or displaying this player, you agree to abide by this license. If you do not agree, do not copy or display the player."